src/app/components/case/tasks/AddTask/scaffold.ts

```diff
@@ -0,0 +1,20 @@
+import { FormPositioner } from "@amsterdam/scaffold-form/package"
+import { Fields } from "app/components/shared/Form/ScaffoldFields"
+
+export default (onChange: (value: string) => void) => {
+
+  const fields = {
+    task: {
+      type: "SelectField",
+      props: {
+        name: "task",
+        options: ["Taak opvoeren", "Opstellen concept aanschrijving"], // TODO use endpoint
+        onChange: onChange
```

Comment:
You could use a property shorthand here. E.g. change `onChange: onChange` => `onChange`

Reply:
Yes, will do